platform/viewer/src/appExtensions/MeasurementsPanel/ConnectedMeasurementTable.js

```diff
@@ -142,8 +162,30 @@ function convertTimepointsToTableData(timepoints) {
   ];
 }
 
+function updateWarnings(measurementCollection) {
+  measurementCollection;
+}
+
+function getSaveFunction(servers) {
+  const activeServer = servers.servers.find(a => a.active === true);
+  let saveFunction = undefined;
+
+  if (activeServer.type === 'dicomWeb') {
+    saveFunction = () => {
+      const measurementApi = OHIF.measurements.MeasurementApi.Instance;
```

Comment:
That's a little wonky. Added to my "we should clean this up" list. Not blocking.

Reply:
👍